core/plugins/org.polarsys.capella.core.projection.scenario/src/org/polarsys/capella/core/projection/scenario/commands/ESToISCommand.java

```diff
@@ -86,10 +84,7 @@ protected AbstractTransform getTransformation(EObject element_p) {
         Scenario scenario = (Scenario) eObject;
         if (postRun || isScenarioValid(scenario)) {
           // add also the scenario referenced by this scenario
-          Set<EObject> referencesScenarios = new HashSet<EObject>();
-          searchReferencedScenarios(scenario, referencesScenarios, result);
-          result.addAll(referencesScenarios);
-          result.add(scenario);
+          addAllScenarios(scenario, result, postRun);
         }
```

Comment:
The test failed because there was the checking valid scenario condition for selecting only one test.
if (postRun || isScenarioValid(scenario))